.circleci/config.yml

```diff
@@ -51,115 +55,102 @@ commands:
             echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
             echo "[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"" >> $BASH_ENV
 jobs:
-  build-linux-node-6:
+  build-linux-node-10:
     docker:
-      - image: circleci/node:6
+      - image: circleci/node:10
     steps:
       - build
       - release
 
-  build-linux-node-7:
+  build-linux-node-11:
     docker:
-      - image: circleci/node:7
+      - image: circleci/node:11
     steps:
       - build
       - release
 
-  build-linux-node-8:
+  build-linux-node-12:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:12
     steps:
       - build
       - release
 
-  build-linux-node-9:
+  build-linux-node-13:
     docker:
-      - image: circleci/node:9
+      - image: circleci/node:13
```

Comment:
I don't think we need Node 13. Let's just stick with the LTS versions -- 12, 14, 16

Reply:
I left those job definitions, but they are not being run. 
https://github.com/bigcommerce-labs/node-sass/pull/33/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L247